resources/assets/less/bem/mp-history-game.less

```diff
@@ -122,6 +138,10 @@
 
     .default-border-radius();
     .default-box-shadow();
+
+    @media @mobile{
+      height: 125px;
```

Comment:
it's because `mp-history-player-score__info-box` sets height at 100% which confused safari